examples/guided.py

```diff
@@ -264,6 +264,12 @@ def perform_installation(mountpoint):
 		# This *can* be done outside of the installation, but the installer can deal with it.
 		installation.mount_ordered_layout(archinstall.storage['disk_layouts'])
 
+		# Placing /boot check during installation because this will catch both re-use and wipe scenarios.
+		for partition in installation.partitions:
+			if partition.mountpoint == installation.target + '/boot':
+				if partition.size <= 0.15: # in GB
```

Comment:
150MB is kind of small for boot. I'd probably do .25 here.

Reply:
Fixed! : )